tests/search/lightgbm/lightgbm.rb

```diff
@@ -15,8 +15,18 @@ def teardown
     stop
   end
 
+  def make_app
+    app = SearchApp.new
+    container = Container.new('default').search(Searching.new)
+    container.component("    <model-evaluation />\n")
+    app.container(container)
+    app.sd(selfdir + 'app/schemas/lightgbm.sd')
+    app
+  end
+
   def test_lightgbm_evaluation
-    deploy(selfdir + "app/")
+    deploy_app(make_app,
```

Comment:
@toregge I guess services.xml and hosts.xml from app is no longer used and can be deleted?